src/main/java/net/sky/pieces/Piece.java

```diff
@@ -32,9 +36,53 @@ public boolean isBlack(Piece piece) {
         return piece.getColor() == Color.BLACK;
     }
 
+    public double getDefaultPoint() {
+        return pieceType.getPoint();
+    }
+
+    public double getPoint(List<Piece> fileList) {
+        double defaultPoint = pieceType.getPoint();
+
+        if (pieceType != PieceType.PAWN) {
+            return defaultPoint;
+        }
+
+        int count = 0;
+        for (Piece piece : fileList) {
+            if (piece.getPieceType() == PieceType.PAWN) {
+                count += 1;
+            }
+        }
+
+        return count > 1 ? defaultPoint / 2 : defaultPoint;
```

Comment:
삼항연산자의 활용 자체는 나쁘지 않은데 개인적인 생각으론 삼항연산자 내부에 연산이 들어가면 그때부턴 가독성이 좀 떨어집니다. 
차라리 아래와 같이 풀어쓰는 것이 가독성 측면에서 더 나을수도 있겠습니다.

```suggestion
        if (count > 1) {
            return defaultPoint / 2;
        }
        return defaultPoint;

Reply:
변경했습니다!